charts/datahub/subcharts/datahub-mce-consumer/values.yaml

```diff
@@ -39,8 +42,14 @@ securityContext: {}
   # runAsUser: 1000
 
 service:
-  type: ClusterIP
-  port: 80
+  type: ClusterIP # NodePort
+  port: "9090"
+  targetPort: actuator
```

Comment:
Why do we have this for mcd-consumer? Mae-consumer does not have it.

Reply:
it already has it. MCE was missing the service for some reason.

Reply:
MAE has the service but in the values.yaml it is using a different port and a different targetPort name, I'm asking if we can make this consistent.

Reply:
renamed it to http for consistency